formPage.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+?>
+
+<!DOCTYPE html>
+<html lang="en-GB">
+
+<head>
+    <title>My Bike Collection</title>
+    <link href="https://fonts.googleapis.com/css?family=M+PLUS+1p|Permanent+Marker&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="Normalize.css">
+    <link rel="stylesheet" type="text/css" href="formStyles.css">
+</head>
+
+<body>
+<section>
+    <h1>Add a bike</h1>
+
+    <div class="addContainer">
+        <div class="buttonContainer">
+            <a href="index.php"><button type="button">BACK TO COLLECTION</button><a/>
+        </div>
+    </div>
+
+    <div class="addBikeForm">
+        <form class="formLeft" method="post" action="functions.php">
```

Comment:
I would recommend not passing to functions.php but a separate php file the user doesn't see that processes the form data

Reply:
Done.